### DISCUSSION TITLE
Solito with Tailwindcss

### DISCUSSION THREAD

**Discussion by adherb** [nandorojo/solito#97] [Q&A]
Has anyone use Solito with TailwindCSS?

I am currently trying to implement it, but I keep getting an import error.

I am using this package: https://www.npmjs.com/package/tailwindcss-react-native

I have added the package to the package.json within the packages folder.

This issue seems to be when I wrap the Next _app.tsx file with the TailwindCSS provider.

Is that correct or should I just be wrapping only the Expo App.tsx with the TailwindCSS provider?

**Comment by nandorojo** [comment]
there’s currently an open pr for this at #72. cc @marklawlor

**Reply by adherb** [reply]
Hey @nandorojo and @marklawlor,

I've been through the PR, I'm pretty new at this so I just though its best to double check with you guys.

Am I right in understanding that TailwindCSS now works with RN using the NativeWind package or does the PR need to be merged first?

**Reply by marklawlor** [reply]
Hey @adherb I've updated the PR to use NativeWind.

Both tailwindcss-react-native and NativeWind work with Solito without the PR. The PR just shows one possible project layout that you can use.

**Reply by adherb** [reply]
Thanks Mark, appreciate the reply and update.
